pyaedt/common_rpc.py

```diff
@@ -376,8 +385,8 @@ def launch_ironpython_server(
 
     Parameters
     ----------
-    aedt_path : str
-        AEDT path on Linux.
+    aedt_path : str, optional
+        AEDT path on Linux. If not provided an ANSYSEM_ROOT2xx has to be setup.
```

Comment:
```suggestion
        AEDT path on Linux. The default is ``None``, in which case an ANSYSEM_ROOT2xx must be set up.
```

Reply:
The added parameters must be reviewed for accuracy. The last one might need an explanation of the behavior that occurs when the default of ``False`` is used. The format would be: The default is ``False``, in which case the explicit path is used. (I have no idea what the behavior would actually be.)